### PR TITLE
express intro completed

### PR DESCRIPTION
**Pull request by NateNobleza**
hello world!

### CODE REVIEW

File: express-intro/server.ts [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+app.listen(8080, () => {
+  console.log('Express server listening on port 8080');
+});
+
+app.use((req, res, next) => {
+  console.log('Hello, World!');
+  console.log('The date is', new Date());
+  next();
+});
```
Comment (Zhiwutian) [inline express-intro/server.ts:13]: Hey Nate, make sure to move this before the `listen` method call. 

the listen method should be at the end of the server

**Review by Zhiwutian — CHANGES_REQUESTED** [review]
one small update!

File: express-intro/server.ts (hunk 1 above)
Comment (NateNobleza) [reply express-intro/server.ts:9]: Screenshot 2024-04-04 121246


the listen function is now at the end

**Review by Zhiwutian — APPROVED** [review]
👍🏼

**Pull request merged**